formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -169,7 +168,7 @@ public class Cbor(
 
         override fun skipBeginToken() = setSize(decoder.startArray())
 
-        override fun decodeElementIndex(descriptor: SerialDescriptor) = if (!finiteMode && decoder.isEnd() || (finiteMode && ind >= size)) READ_DONE else ind++
+        override fun decodeElementIndex(descriptor: SerialDescriptor) = if (!finiteMode && decoder.isEnd() || (finiteMode && ind >= size)) CompositeDecoder.DECODE_DONE else ind++
```

Comment:
why not import this constant?

Reply:
This is an automatic replacement.
Replacing just with `ReplaceWith("DECODE_DONE ")` isn't working in IDEA in corner cases.
Fixed